include/boost/json/parse.hpp

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright (c) 2020 Krystian Stasiowski (sdkrystian@gmail.com)
```

Comment:
hey now, I wrote everything in this file. moving it to a new file doesn't mean you have the sole authorship

Reply:
omg sorry I'll change that